livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -180,6 +215,9 @@ protected List<String> parseSQLJsonOutput(String output) {
     List<String> rows = new ArrayList<>();
 
     String[] rowsOutput = output.split("(?<!\\\\)\\n");
+    if(rowsOutput.length <= 1){
```

Comment:
Could you add a comment to explain what this case mean?

Reply:
> Could you add a comment to explain what this case mean?

message.getData() for DDL statement is only 1 line : "df: org.apache.spark.sql.DataFrame = []"

if continue , `String[] header = rowsOutput[1].split("\t");` will throw out of index exception, this is a bug in previous version.

comment has been added in the next commit.